src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -22,23 +30,36 @@
 
 public class CreateRequestService {
 
+  private static Map<RequestType, NoticeEventType> requestTypeToEventMap;
```

Comment:
I think I may have described my feedback poorly.

I wasn't suggesting inline the request patron notice sender into the create request service. Only to move the invocation of it from outside of updating the loan.

Why was the decision made to inline the request sender?

Reply:
There is no special reason for that, I'm mainly ok with both options. Reverted last changes, so that `RequestNoticeSender` is called from `CreateRequestService` and 'UpdateRequestService'.